src/page.rs

```diff
@@ -14,23 +14,22 @@ impl PopplerPage {
         let mut width: f64 = 0.0;
         let mut height: f64 = 0.0;
 
-        unsafe {
-            sys_pg::poppler_page_get_size(
-                self.0,
-                &mut width as *mut f64 as *mut c_double,
-                &mut height as *mut f64 as *mut c_double,
-            )
+        {
+            let width = (&mut width as *mut f64) as *mut c_double;
+            let height = (&mut height as *mut f64) as *mut c_double;
+
+            unsafe { sys_pg::poppler_page_get_size(self.0, width, height) }
         }
 
         (width, height)
     }
 
-    pub fn render(&self, ctx: &cairo::Context) {
+    pub fn render(&self, ctx: &mut cairo::Context) {
```

Comment:
the `to_raw_none` returns a `*mut` (rust analyzer shows that ^^), so I figured it's safer to also require an `&mut` in the interface